### DISCUSSION TITLE
CSS with woff2 only - no woff

### DISCUSSION THREAD

**Discussion by cw789** [fontsource/fontsource#574] [Q&A] [answered]
Quick ask if I oversee something.
I think it is not possible to import the CSS without the legacy woff fallback - only getting woff2. Am I right?

I personally think having the legacy woff as fallback isn't worth it anymore.

**Comment by ayuhito** [comment, marked as answer]
At the moment no. We don't have any plans on deprecating WOFF files anytime soon.

**Reply by KTibow** [reply]
Has this changed? By now, most browsers have supported woff2 for 11 years and all for 5. It would also prevent nasty situations where the woff is inlined so it has to be downloaded even if unwanted.

**Reply by ayuhito** [reply]
The next major release sometime this year will drop it, yes: https://github.com/fontsource/fontsource/issues/1068
